plugin/RealmPlugin/RLMPSimulatorManager.m

```diff
@@ -0,0 +1,95 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMPSimulatorManager.h"
+
+#import "NSTask+LaunchAdditions.h"
+
+static NSString * const RLMPBootedSimulatorKey = @"Booted";
+
+@interface RLMPSimulatorManager ()
+
+@end
+
+@implementation RLMPSimulatorManager
+
++ (NSString *)bootedSimulatorUUID {
+    
+    NSString *deviceData = [self readDeviceData];
+    
+    __block NSString *bootedDeviceUUID;
+    if (deviceData) {
+        // Process output
+        NSDictionary *deviceStatuses = [self processDeviceData:deviceData];
+        
+        [deviceStatuses enumerateKeysAndObjectsUsingBlock:^(NSString *key, NSString *value, BOOL *stop) {
+            if ([value isEqualToString:RLMPBootedSimulatorKey]) {
+                bootedDeviceUUID = key;
+                // Stop when we found single booted device
+                *stop = YES;
+            }
+        }];
+    }
+    
+    return bootedDeviceUUID;
+}
+
++ (NSString *)readDeviceData {
+    // Set parameters to get device detail
+    NSString *path = @"/usr/bin/xcrun";
```

Comment:
This could potentially get the iOS Simulator directories from a different version of Xcode than what's currently running if another version is set via `xcode-select`